captcha.module

```diff
@@ -311,18 +311,20 @@ function captcha_form_alter(array &$form, FormStateInterface $form_state, $form_
       if ($config->get('add_captcha_description')) {
         $captcha_element['#description'] = _captcha_get_description();
       }
+      if ($captcha_point->status()) {
+        // Get placement in form and insert in form.
+        $captcha_placement = _captcha_get_captcha_placement($form_id, $form);
+        _captcha_insert_captcha_element($form, $captcha_placement, $captcha_element);
+      }
```

Comment:
Why not do the check earlier, in the existing if ($captcha) check? Do the tests except to enable it again from there? This will then still show description and so on, which is a bit weird?